src/main/java/org/cascadebot/cascadebot/commands/moderation/BanCommand.java

```diff
@@ -56,8 +56,7 @@ public void onCommand(Member sender, CommandContext context) {
                     "forceban_user",
                     context.getChannel(),
                     MessageType.DANGER,
-                    "**We couldn't find that user in this guild!** \n" +
-                            "If you would like to forcefully ban them, please react to this message!",
+                    context.i18n("commands.ban.forcefully_ban?"),
```

Comment:
I don't think having special charaters in lang names is a good idea as it can cause confusion and miss typing.

Reply:
By special characters you mean the `?` in this case?

Reply:
Yep

Reply:
Agreed